__tests__/Unit/pages/Standup/Standup.test.tsx

```diff
@@ -0,0 +1,92 @@
+import StandUp from '@/pages/standup';
+import { renderWithRouter } from '@/test_utils/createMockRouter';
+import { Provider } from 'react-redux';
+import { store } from '@/app/store';
+import * as AuthenticatedHooks from '@/hooks/useAuthenticated';
+import {
+    userLoggedIn,
+    userNotLoggedIn,
+    notLoading,
+} from '../../../../__mocks__/db/standup';
+
+describe('Standup', () => {
+    test('should be able to render standup component if user is  logged-in and authenticated', () => {
+        jest.spyOn(AuthenticatedHooks, 'default').mockImplementation(
+            () => userLoggedIn
+        );
+        const { getByText } = renderWithRouter(
+            <Provider store={store()}>
+                <StandUp />
+            </Provider>,
+            {
+                query: { dev: 'true' },
+            }
+        );
+        expect(getByText('Loading...')).toBeInTheDocument();
+        expect(getByText('Standup')).toBeInTheDocument();
```

Comment:
I think this test will test if there is an option in breadcrumb for standup.

Reply:
what that does mean, and is there anything for me to fix?

Reply:
I cant see any test where it says should not render component when dev true is not passed.

Reply:
added test case